app/models/photo.rb

```diff
@@ -32,19 +32,30 @@ class Photo < ActiveRecord::Base
         id: photo.status_message.id
       } if photo.status_message
     }, as: :status_message
+    t.add lambda { |photo|
+      {
+		address: photo.location.address,
```

Comment:
- Tab detected.
- Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.